src/main/webapp/data-visualization.js

```diff
@@ -306,38 +301,24 @@ function setStyle(isCountyQuery) {
   mapsDiv.style.display = 'none';
 }
 
-function initDataTable(isCountyQuery) {
-  const table = document.createElement('table');
-  table.innerHTML = '';
-  const tableRow = document.createElement('tr');
-  const region = document.createElement('th');
-  const value = document.createElement('th');
-  region.innerText = isCountyQuery ? 'County' : 'State';
-  value.innerText = 'Population';
-  tableRow.appendChild(region);
-  tableRow.appendChild(value);
-  table.appendChild(tableRow);
-  return table;
-}
-
-/**
- * Create and return an HTML table using the data in dataArray.
- * Uses first row in dataArray as table headers.
- */
-function appendRowToDataTable(table, locationName, population) {
-  const tableRow = document.createElement('tr');
-  const region = document.createElement('td');
-  const value = document.createElement('td');
-  region.innerText = locationName;
-  value.innerText = population;
-  tableRow.appendChild(region);
-  tableRow.appendChild(value);
-  table.appendChild(tableRow);
+// Draw data table using Visualization API
+function drawTable(dataArray, isCountyQuery) {
+  google.charts.load('current', {'packages': ['table']});
+  google.charts.setOnLoadCallback(() => {
+    const data = new google.visualization.DataTable();
+    const nameHeader = isCountyQuery ? 'County' : 'State';
+    data.addColumn('string', nameHeader);
+    data.addColumn('number', 'Population');
+    dataArray.forEach((elem) => {
+      data.addRow([elem.name, parseInt(elem.value)]);
+    });
+    const table = new google.visualization.Table(
+        document.getElementById('data-table'));
+    table.draw(data, {width: '100%', height: '100%'});
```

Comment:
Do we want to reduce the width of the table? It stretches across my screen but it's mostly empty space, and the numbers and the text end up really far away from each other. I think it might read more easily at approximately the width you had before

Reply:
This may be a hot take but I think I actually like it better without the data table API. Maybe we could adjust the width/font size of this table to make it look nicer but I'm still not a fan of the header in this one. Happy to hear any other thoughts though

Reply:
I agree that the API makes it look super weird. But maybe there is room for adjustment? Like to make the header not have that weird gradient?

Reply:
The visualization API allows configuring the table with any styles we'd like, for now I just removed the header gradient. Also may the width 30%